liquibase-core/src/test/groovy/liquibase/parser/core/xml/XMLChangeLogSAXParser_RealFile_Test.groovy

```diff
@@ -281,7 +282,16 @@ public class XMLChangeLogSAXParser_RealFile_Test extends Specification {
         e.message.startsWith("The file ${path} was not found")
     }
 
-    def "ChangeLogParseException thrown if changelog has invalid tags"() throws Exception {
+    def "ChangeLogParseException is thrown if the changeset has invalid tags"() throws Exception {
+        when:
+        def d = new XMLChangeLogSAXParser().parse("liquibase/parser/core/xml/malformedChangeLog.xml", new ChangeLogParameters(), new JUnitResourceAccessor())
```

Comment:
Is the variable `d` necessary? It looks like maybe it was used for debugging?

Reply:
I forgot it there, thanks. Fixed!